include/openmc/constants.h

```diff
@@ -358,6 +358,10 @@ constexpr int CMFD_NOACCEL {-1};
 
 enum class GeometryType { CSG, DAG };
 
+//==============================================================================
+// Variance Reduction constants
+constexpr double WEIGHT_CUTOFF {1.0e-38}; // default low weight cutoff
```

Comment:
IMO we should put module-specific constants into module-specific headers, in this case the weight window one. It's just better-organized that way. Suppose OpenMC continues to grow for the next ten years. Once people add muon physics and all sorts of other random stuff, the constants.h file will be a mile long.

Reply:
I agree with this -- let's move it to `weight_windows.h`. From a practical perspective, changing this value when it's in constants.h would result in recompilation in most of the code, which doesn't make sense.

Reply:
I'm a little confused because this would break our typical protocol for placing these types of hardcoded values in `constants.h`, even if they are only used in one source file (i.e. `MAX_EVENTS`, `FREE_GAS_THRESHOLD`). And while it's true that most of the code would have to recompile if this `DEFAULT_WEIGHT_CUTOFF` is changed, most likely this value will be changed based on the parameters read from the `settings.xml` by the `WeightWindowSettings` class rather than changing the default cutoff value in `constants.h`.

Reply:
I would be in favor (as would @gridley it sounds like) of abandoning that protocol and moving constants closer to their point of use. But, I suppose it is better to leave that for another day and just do it all at once if we're going to move them around.

Reply:
Ah I see. I'm on board. I've created another issue regarding this and will follow up on it after this gets merged in.